packages/core/strapi/lib/services/entity-validator/__tests__/email-validators.test.js

```diff
@@ -37,5 +37,18 @@ describe('Email validator', () => {
 
       expect(await validator('valid@email.com')).toBe('valid@email.com');
     });
+
+    test('it validates non-empty email required field', async () => {
+      const validator = strapiUtils.validateYupSchema(
+        validators.email(
+          {
+            attr: { type: 'string', required: true },
+          },
+          { isDraft: false }
+        )
+      );
+
+      expect(await validator('valid@email.com')).not.toBe('');
```

Comment:
I'm not sure I understand the test here. Shouldn't you check that sending an empty string throws?

Reply:
Hi @alexandrebodin,
sorry first time with tests. I pushed a fix, can you review it?